kornia/augmentation/container/augment.py

```diff
@@ -331,7 +337,12 @@ def forward(  # type: ignore[override]
         label: Optional[Tensor] = None,
         params: Optional[List[ParamItem]] = None,
         data_keys: Optional[List[Union[str, int, DataKey]]] = None,
-    ) -> Union[Tensor, List[Tensor], Tuple[Union[Tensor, List[Tensor]], Optional[Tensor]]]:
+    ) -> Union[
+        Tensor,
+        List[Tensor],
+        List[Union[Tensor, List[Tensor]]],
+        Tuple[Union[Tensor, List[Tensor], List[Union[Tensor, List[Tensor]]]], Optional[Tensor]],
```

Comment:
What is this one?

Reply:
because of the return of `__packup_output__` can be a tuple when pass a label

Reply:
Can you help remove the `label` argument? #2112 Then we may have a simpler return types.

I have refactored `MixAugmentations` already. It was indeed a bad design to include labels in forward arguments.

Reply:
Do it in another PR. I will merge this one.